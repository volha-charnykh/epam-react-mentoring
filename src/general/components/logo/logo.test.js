import React from 'react';
import { render } from '@testing-library/react';
import Logo from './logo';

describe('Logo', () => {
  it('renders Logo snapshot', () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
