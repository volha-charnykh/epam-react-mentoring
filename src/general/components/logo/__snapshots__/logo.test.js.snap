// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders Logo snapshot 1`] = `
<DocumentFragment>
  <span
    class="Logo"
  >
    <span
      class="LogoBold"
    >
      netflix
    </span>
    roulette
  </span>
</DocumentFragment>
`;
